CHANGELOG.md

```diff
@@ -1,11 +1,11 @@
 ## 1.0.5-wip
```

Comment:
I think we could as easily rev to `1.0.5` here and publish; the automation is hooked up for this repo.